test/e2e/specs/migrate_gitops_provv1_mgmtv3.go

```diff
@@ -362,7 +361,7 @@ func MigrateToV3UsingGitOpsSpec(ctx context.Context, inputGetter func() MigrateT
 			BootstrapClusterProxy:        input.BootstrapClusterProxy,
 			HelmBinaryPath:               input.HelmBinaryPath,
 			Namespace:                    turtlesframework.DefaultRancherTurtlesNamespace,
-			Image:                        fmt.Sprintf("ghcr.io/rancher/turtles-e2e-%s", runtime.GOARCH),
```

Comment:
Just to confirm: I remember changing this in https://github.com/rancher/turtles/pull/187/commits/53df8418395c62281870c8004e3d99dccf24b542 when e2e images were not available locally for Mac, but this change should not affect since we are removing arch completely from image building Makefile?

Reply:
you're right, it will work on all platforms